test/integration/scheduler_perf/util.go

```diff
@@ -75,6 +76,8 @@ func newDefaultComponentConfig() (*config.KubeSchedulerConfiguration, error) {
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
 func mustSetupScheduler(config *config.KubeSchedulerConfiguration) (util.ShutdownFunc, coreinformers.PodInformer, clientset.Interface, dynamic.Interface) {
+	// Run API server with minimimal logging.
+	framework.MinVerbosity = 0
```

Comment:
So we cannot longer control this via a command line flag?

Reply:
This is a minimum, not the final value. In `test/integration/scheduler_perf` all values for `-v` work as expected, i.e. `-v=1` runs at verbosity 1.

It's everywhere else that `-v=1` gets bumped up to 4 when the apiserver is started. This seemed odd to me, but apparently was desired.